aspnetcore/security/enforcing-ssl/includes/enforcing-ssl7.md

```diff
@@ -0,0 +1,1182 @@
+:::moniker range="> aspnetcore-6.0 < aspnetcore-8.0"
```

Comment:
How is this different from `= aspnetcore-7.0`?

Reply:
It shouldn't be, I'm trying it because with the "=" alone I got build errors saying I had duplicate H2 sections, which usually is because of an accidental overlap. Moniker ranges sometimes act unpredictably.